app/components/Form/ApplicationForm.tsx

```diff
@@ -357,9 +381,22 @@ const ApplicationForm: React.FC<Props> = ({
       return;
     }
 
-    if (isAcknowledgementPage)
+    if (isAcknowledgementPage) {
       updateAreAllAcknowledgementFieldsSet(newFormSectionData);
-    if (isSubmitPage) updateAreAllSubmissionFieldsSet(newFormSectionData);
+    }
+    if (isSubmitPage) {
+      updateAreAllSubmissionFieldsSet(newFormSectionData);
+    }
+    if (isProjectAreaPage) {
+      const projectAreaAccepted = acceptedProjectAreasArray.includes(
+        newFormSectionData?.geographicArea[0]?.toString()
+      );
+      setProjectAreaOpen(!projectAreaAccepted);
+      setProjectAreaModalOpen(
```

Comment:
I'm not sure if this is intended behaviour, but right now anytime someone clicks on a form element in the project area page while a zone that isn't for this intake is chosen you get the modal popping up every time. 

Click zone 7 -> modal opens -> close modal -> click "yes" for project spanning multiple territories -> modal opens again

Reply:
Also I don't know if this was introduced in this PR, but right now I'm only able to select 1 zone at a time.

Reply:
great catch, working on the fix. 

as for the second question, yes, that was handled as part of a different ticket. new intakes will only allow you to select one project area at a time.

Reply:
Ahh gotcha